dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -231,6 +231,15 @@ def _do_aggregations(
         for col in group_columns:
             collected_aggregations[None].append((col, col, "first"))
 
+        groupby_agg_options = context.schema[
+            context.schema_name
```

Comment:
I was giving this a bit more thought and it seems like `context.schema_name` always returns the default schema, and the `context.fqn` method can help return the specific schema being used in that query. I'm not completely sure if the schema for groupby is associated with the groupby_call itself or with specific aggregations similar based on the code here: https://github.com/dask-contrib/dask-sql/blob/5b8f8a927fbd8416e52c154f6b4f680b0553fd55/dask_sql/physical/rel/logical/aggregate.py#L285-L287